app/app/src/main/java/me/bickositieff/raspio/ui/playlist/PlaylistFragment.kt

```diff
@@ -0,0 +1,39 @@
+package me.bickositieff.raspio.ui.playlist
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.TextView
+import androidx.fragment.app.Fragment
+import androidx.lifecycle.ViewModelProvider
+import androidx.recyclerview.widget.RecyclerView
+import me.bickositieff.raspio.R
+
+class PlaylistFragment : Fragment() {
+
+    private lateinit var playlistViewModel: PlaylistViewModel
```

Comment:
Could have been initialized using `by viewModel()` (see https://stackoverflow.com/q/56748334/8719698) but results in the same